Sources/FeaturevisorSDK/bucket.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+let HASH_SEED: UInt32 = 1
+let MAX_HASH_VALUE = UInt32.max // 2^32 - 1
+let MAX_BUCKETED_NUMBER = 100000 // 100% * 1000 to include three decimal places in the same integer value
+
+func getBucketedNumber(bucketKey: String) -> Int {
+  var hasher = Hasher()
```

Comment:
in future, we need to make sure that it uses MurmurHash algorithm for the hashing, to maintain consistent bucketing across different SDKs in different languages.

test cases with some example bucket keys are found here: https://github.com/fahad19/featurevisor/blob/main/packages/sdk/src/bucket.spec.ts#L22-L29

just for reference, including for myself.

Reply:
Very good point, wasn't thinking about cross-device bucketing. It should be fixed now